readme.md

```diff
@@ -824,3 +824,6 @@ Afterwards apply the correct permissions with `sudo chown root:root /root/automa
 1. Open the cronjob with `sudo crontab -u root -e` (and choose your editor of choice if not already done. I'd recommend nano). 
 1. Add the following new line to the crontab if not already present: `0 5 * * * /root/automatic-updates.sh` which will run the script at 05:00 each day. 
 1. save and close the crontab (when using nano the shortcuts for this are `Ctrl + o` then `Enter` to save, and close the editor with `Ctrl + x`).
+
+### Securing the AIO interface from unauthorized ACME challenges
+(By design)[https://github.com/nextcloud/all-in-one/discussions/4882#discussioncomment-9858384], Caddy that runs inside the mastercontainer, which handles automatic SSL certificate generation for the AIO interface, is vulnerable to receiving DNS challenges for arbitrary hostnames from anyone on the internet. While this does not compromise your server's security, it can result in cluttered logs and rejected certificate renewal attempts due to rate limit abuse. To mitigate this issue, it is recommended to place the AIO interface behind a VPN and/or limit its public exposure.
```

Comment:
> Caddy that runs inside the mastercontainer, (...), is vulnerable to receiving DNS challenges

That statement is not accurate on 2 accounts:
- Caddy itself is not vulnerable. The configuration that's shipped by Nextclout-AIO is. If the user explicitly configures their server in a vulnerable way, the vulnerability can't be blamed on the server. Caddy ships with guardrails (not enabling `on_demand` by default, and the `ask` URL being mandatory), but the config author explicitly overrode them through the custom PHP script.
- The challenge seen in the logs is HTTP-01 challenge, not DNS challenge. The difference is critical because it's the HTTP and ALPN challenges that rely on `on_demand`. The attacker doesn't have to abuse DNS at all -- they can craft requests with fake `Host` header/SNI -- but by default, Caddy would ignore/reject those requests due to not being known domain names.